docs/en/clever-show.md

```diff
@@ -0,0 +1,18 @@
+# clever-show
+
+Software for making the drone show controlled by Raspberry Pi, PX4 and COEX [Clever](https://github.com/CopterExpress/clever) package.
+
+Create animation in Blender, convert it to drone paths, set up the drones and run your own show!
+
+Project repository: https://github.com/CopterExpress/clever-show
+
+## Demo video
+
+<iframe width="966" height="543" src="https://www.youtube.com/embed/HdHbZFz7nR0" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+12 drones perform in a show in Electrotheatre Stanislavsky, Moscow.
+
+## Team
+
+* Arthur Golubtsov, software engineer COEX, https://github.com/goldarte
+* Artem Vasyunik, software engineer intern COEX, https://github.com/artem30801
```

Comment:
software engineer in COEX, software engineer intern in COEX?

Reply:
Updated